examples/web-wagmi/src/publications/UsePublications.tsx

```diff
@@ -62,6 +64,12 @@ export function UsePublications() {
           ))}
         </fieldset>
 
+        {beforeCount > 0 && (
+          <button disabled={loading} onClick={prev}>
```

Comment:
I'm not sure without running it locally and debugging, does `loading` flag switches to `true` when you call `prev`?

Reply:
mmmhh.. good question... in my tests turns out it wasn't even doing the request... it was using the cache... will test

Reply:
checked... `loading` is stable!